src/prob/common.jl

```diff
@@ -0,0 +1,7 @@
+
+function run_mc_model(data::Dict{String,<:Any}, model_type::Type, solver, build_mc::Function; ref_extensions::Vector{<:Function}=Vector{Function}([]), make_si=!get(data, "per_unit", false), multinetwork::Bool=false, kwargs...)::Dict{String,Any}
+    result = _PM.run_model(data, model_type, solver, build_mc; ref_extensions=[_PMD.ref_add_arcs_transformer!, ref_extensions...], multiconductor=true, multinetwork=multinetwork, solution_processors=[solution_fs!], kwargs...)
+    result["solution"] = _PMD.transform_solution(result["solution"], data; make_si=make_si)
+    add_solution!(result["solution"], data)
+    return result
+end
```

Comment:
is there a reason you can't use the `run_mc_model` from PMD, is it the `solution_processors` part, if so you should be able to pass that as a kwarg without problem (otherwise that's a bug that should get fixed in PMD)